src/repl.ts

```diff
@@ -230,6 +231,19 @@ function enterPasteMode() {
   })
 }
 
+function loadFile(filename: string) {
+  try {
+    console.log(`// loading file ${filename}`.cyan)
```

Comment:
I think it should. Also I think **:load** command should accept a list of files instead of just one.